### ISSUE TITLE
[FEATURE] Don't restart whole node after lnd.conf change.

### ISSUE THREAD

**Issue by Wired4ncer** [mynodebtc/mynode#657]
Changes to `lnd.con`f always trigger reboot of whole node, which is not necessary and should not be enforced. This excessive and long restarts reduce uptime of the node significantly.

You could add option in gui to "**Restart node now**" or "**Restart later**". Some of the changes doesn't need to get restart immediately or could be scheduled for next restart. Even better approach would be to restart only `lnd.service` rather than whole node. 

Please consider this change as any more serious routing node need to keep uptime as high as possible.

Thank you

**Comment by tehelsper** [comment]
What settings require frequent changing? Some apps in the past have not worked well if lnd restarts underneath them, so I set it to do a full reboot. I also thought the config would rarely need updating. Is that not the case?

**Comment by Wired4ncer** [comment]
It is not frequent, but it does hurt uptime even if done once per month / two months. Full node restart takes anywhere from 10-20 minutes with ssd drive. I have seen that other LN node implementation only restart service. 

Or if you feel like restarting service cause issues why not make the first option to let user to choose whether he wants node to restart to apply changes or just wait for when the node is restarted? That would not cause any problems would it?

Basically now any change in lnd cause uptime loss even if right after I will be doing upgrade which will require another restart. I would happily schedule the changes prior to upgrade if that would be option.

**Comment by Wired4ncer** [comment]
We have tried to avoid restarting by updating lnd.conf in terminal, but you have there some script which delete all the changes and set up the last state as per web gui.

**Comment by tehelsper** [comment]
Oh yeah, if you want to edit it manually you definitely can. It's stored here:

/mnt/hdd/mynode/settings/lnd_custom.conf

Then you could restart LND either on the application page or via this command:

sudo systemctl restart lnd

**Comment by Wired4ncer** [comment]
Perfect, this is exactly what I needed. Thank you. It could still be beneficial for people to have the option not to restart right after the lnd.conf change in gui, but for my purpose I'm more than happy to use terminal. I will write those steps to mynode.community for others to know. Will close the issue for now. Thank you for clarification.
